### ISSUE TITLE
Add ISO codes to the estimate output data

### ISSUE THREAD

**Issue by hnykda** [epidemics/covid#253]
Currently, ISO codes are used to match estimates to the geomap. Putting ISO codes in our data permanently would solve this, but it has a downside of having problem mapping cities to it (because they don't have a code). We could use any other key (in theory) from the [geojson](https://storage.googleapis.com/static-covid/static/data/casemap-geo.json). Nevertheless, having ISO codes is probably a good start and according to Tomas probably not that hard.

@gavento can you do it as per https://epidemicforecasting.slack.com/archives/C0100BY1EJZ/p1585060100006900?thread_ts=1584983742.000700&cid=C0100BY1EJZ please?

**Comment by gavento** [comment]
@hnykda Sorry, I won't get to it (basically, I need to get some sleep, other things took longer).

I understood that you already have an ipy NB with it - could you, pretty please, just add it to the region.yaml file?

**Comment by hnykda** [comment]
I have only a subset of countries, not cities for example, so I was hesitant to put it in the region file. Should I do it anyway?

**Comment by gavento** [comment]
I added the ISO codes to DB (except some ~10 smallest countries) and to the output. Switched the case-map to load the main file. Deployed in master now.

**Comment by hnykda** [comment]
Thanks @gavento . As you can see in the timeline, I gave this task to @veselatereza who spend some hour trying to understand where to add it, wanted to test it, so wasted work now :confused: .

Also, the direct commits to master are making it quite hard to follow the changes...

**Comment by gavento** [comment]
@veselatereza Thanks for coming into the project! And apologies for overtaking you - I missed the status change. (We really needed it yesterday to have updated data in the map, along with more countries.)
